aibolit/patterns/assert_in_code/assert_in_code.py

```diff
@@ -0,0 +1,46 @@
+import javalang.ast
+import javalang.parse
+import javalang.tree
+
+
+_TEST_CLASS_SUFFIX = 'Test'
+
+
+class AssertInCode(object):
+    def value(self, filename: str):
+        tree = self.__file_to_ast(filename)
+
+        return self.__traverse_node(tree)
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
```

Comment:
@tribals Just for you to know, this method is repeated in many places, so I consider it as a copy-paste, that's why I created a ticket https://github.com/yegor256/aibolit/issues/131 with a suggestion to improve that case.

Reply:
I agree. We need to invent some kind of interface/base class for patterns. Do you suppose that refactoring of already existing patterns should be done as part of this issue?

Reply:
@tribals No, this is just for you to know.